go-chaos/cmd/cluster.go

```diff
@@ -69,14 +69,17 @@ func AddClusterCommands(rootCmd *cobra.Command, flags *Flags) {
 	clusterCommand.AddCommand(scaleCommand)
 	scaleCommand.Flags().IntVar(&flags.brokers, "brokers", 0, "The amount of brokers to scale to")
 	scaleCommand.Flags().Int32Var(&flags.replicationFactor, "replicationFactor", -1, "The new replication factor")
-	scaleCommand.MarkFlagRequired("brokers")
+	scaleCommand.Flags().Int32Var(&flags.partitionCount, "partitionCount", -1, "The number of partitions to scale to")
 	forceFailoverCommand.Flags().Int32Var(&flags.regions, "regions", 1, "The number of regions in the cluster")
 	forceFailoverCommand.Flags().Int32Var(&flags.regionId, "regionId", 0, "The id of the region to failover to")
 	forceFailoverCommand.MarkFlagRequired("regions")
 	forceFailoverCommand.MarkFlagRequired("regionId")
 }
 
 func scaleCluster(flags *Flags) error {
+	if flags.partitionCount <= 0 && flags.brokers <= 0 && flags.replicationFactor <= 0 {
```

Comment:
🔧 We can use `.MarkFlagsOneRequired()`

Reply:
:facepalm: just noticed it as well :laughing: